src/Parser/Core/Modules/ManaValues.js

```diff
@@ -26,9 +37,29 @@ class ManaValues extends Analyzer {
             max: max,
             used: manaCost,
           });
+          // The variable 'max' is constant but can differentiate by racial/items.
+          this.baseMana = max;
         });
     }
   }
+
+  suggestions(when) {
+    if(this.combatants.selected.spec.role !== ROLES.HEALER) {
+      return;
+    }
+    const manaPercentageLeft = this.endingMana/this.baseMana;
+
+    when(manaPercentageLeft).isGreaterThan(this.constructor.SUGGESTION_MANA_BREAKPOINT)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>You ended the fight with too much mana. If possible, try to deplete all your mana when the boss dies. A good rule of thumb is having the same mana % as the bosses health %.
```

Comment:
`You ended the fight with too much mana` How about `You had mana left at the end of the fight.`? Technically you can never have too much mana.

Also I'd prefer just writing out *percentage* instead of %, or how about "levels"?